e2e/suite_test.go

```diff
@@ -147,11 +150,11 @@ func bootstrapVMSS(ctx context.Context, t *testing.T, r *mrand.Rand, opts *scena
 	}
 
 	vmssName := fmt.Sprintf("abtest%s", randomLowercaseString(r, 4))
-	t.Logf("vmss name: %q", vmssName)
+	log.Printf("vmss name: %q", vmssName)
 
 	cleanupVMSS := func() {
-		t.Log("deleting vmss", vmssName)
-		poller, err := opts.cloud.vmssClient.BeginDelete(ctx, *opts.chosenCluster.Properties.NodeResourceGroup, vmssName, nil)
+		log.Println("deleting vmss", vmssName)
```

Comment:
can we be consistent on Println with multiple args vs. Printf? I'd personally lean towards Printf but that's just me

Reply:
fixed, switched log.Println calls where there were extra args to log.Printf